toothpick-compiler/src/main/java/toothpick/compiler/common/ToothpickProcessor.java

```diff
@@ -193,20 +215,35 @@ protected void readOptionAnnotationTypes() {
     }
   }
 
+  private void readOptionCrashWhenMethodIsNotPackageProtected() {
+    Map<String, String> options = processingEnv.getOptions();
+    if(toothpickCrashWhenMethodIsNotPackageVisible == null) {
+      toothpickCrashWhenMethodIsNotPackageVisible = Boolean.parseBoolean(options.get(PARAMETER_CRASH_WHEN_INJECTED_METHOD_IS_NOT_PACKAGE));
+    }
+  }
```

Comment:
we are doing it for both processor, but we only listen to the annotation on the MI?